internal/metrics/command.go

```diff
@@ -24,6 +25,17 @@ type commandStats struct {
 	FlapsDuration   float64 `json:"fd"`
 }
 
+type contextKey struct{}
+
+// Adds the InstanceID to context
+func NewContext(ctx context.Context) context.Context {
+	return context.WithValue(ctx, contextKey{}, ulid.Make().String())
+}
+
+func InstanceIDFromContext(ctx context.Context) string {
```

Comment:
`InvocationId` seems like a better name than `InstanceId` since instance is somewhat overloaded. You also use it as the variable name when calling this. Thoughts?

Reply:
Agreed, I'll change that now.